src/OpenLoco/src/Map/MapGenerator/MapGenerator.cpp

```diff
@@ -441,16 +442,63 @@ namespace OpenLoco::World::MapGenerator
         call(0x004597FD, regs);
     }
 
+    template<typename Func>
+    static void generateMiscBuilding(const BuildingObject* buildingObj, const size_t id, Func&& predicate)
+    {
+        uint8_t randomComponent = getGameState().rng.randNext(0, buildingObj->var_9F / 2);
```

Comment:
is it possible to name `var_9F` now?

Reply:
Something like `baseNumberOnMap`, perhaps?

Reply:
yeah `baseNumberOnMap` seems fine, the end results seems to be a random number in the range
`[0.75, 1.25]` * `var_9F`, so the median/average will be `var_9F`

Reply:
Yep, that's pretty much it. Could call var_9F the mean, with .25 being the std.dev?